components/content_settings/core/browser/brave_content_settings_pref_provider_unittest.cc

```diff
@@ -733,4 +786,41 @@ TEST_F(BravePrefProviderTest, EnsureNoWildcardEntries) {
   provider.ShutdownOnUIThread();
 }
 
+TEST_F(BravePrefProviderTest, TestShieldsHttpsMigration) {
+  {
+    BravePrefProvider pref_provider(
+        testing_profile()->GetPrefs(), false /* incognito */,
+        true /* store_last_modified */, false /* restore_session */);
+    DefaultProvider default_provider(testing_profile()->GetPrefs(), false);
+
+    ShieldsHTTPSESetting shields_https_settings(&pref_provider,
+                                                &default_provider);
+
+    shields_https_settings.SetPreMigrationDefaultSetting(
```

Comment:
using a method here actually makes things much harder to debug when there are issues. It's also easier to understand and read inline.

Reply:
done, thanks!